apps/data-scanner/lib/db/migrations/sqls/20220402120235-add-statuses-table-up.sql

```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS statuses;
+CREATE TABLE statuses(
+	status_id SERIAL PRIMARY KEY,
+	full_name VARCHAR (50),
+	username VARCHAR (30),
+	was_online VARCHAR (30),
```

Comment:
was_online and check_date should be a timestamp data type in other case usage of these table will be very hard

Reply:
done